### PR TITLE
cdc_*: delete some code that use old owner

### PR DESCRIPTION
**Pull request by asddongmen**
### What problem does this PR solve? 

Delete some code that that relied on the old leader.

### Check List 

Tests 

 - No code

Code changes

 - Has exported function/method change
 - Has exported variable/fields change
 - Has interface methods change
 - Has persistent data change

Side effects

 - None

Related changes

 - Need to cherry-pick to the release branch

### Release note

- No release note

### CODE REVIEW

**Comment by amyangfei** [comment]
Should we cherry-pick to release-4.0

**Comment by ben1009** [comment]
seems we should have the new owner tested in 5.1 first, then pick to 5.0, could u help confirm @overvenus ?

**Comment by asddongmen** [comment]
> Should we cherry-pick to release-4.0
It seems that the new owner is not opened in 4.0 & 5.0 branch. 
So we can cheery-pick it to the 5.1 branch first, and we can manually cherry-pick it to 4.0 & 5.0 branches later.

File: cdc/server.go [hunk 1]
```diff
@@ -56,11 +53,7 @@ const (
 
 // Server is the capture server
 type Server struct {
-	captureV2 *capture.Capture
-
-	capture      *Capture
-	owner        *Owner
-	ownerLock    sync.RWMutex
+	captureV2    *capture.Capture
```
Comment (overvenus) [inline cdc/server.go:56]: Could you rename 'captureV2' to 'capture'?

**Review by overvenus — COMMENTED** [review]
Rest LGTM

File: cdc/server.go (hunk 1 above)
Comment (asddongmen) [reply cdc/server.go]: done

**Review by overvenus — APPROVED** [review]

**Review by amyangfei — APPROVED** [review]

**Pull request merged**